traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/BookingItemSearchParameters.java

```diff
@@ -86,7 +86,10 @@ public BookingItemSearchParameters instantiate(String rawResponse) throws Except
 
             List<BookingItemCategory> categories = new ArrayList<>();
             if (!jsonObject.isNull("subCategories")) {
-                categories = new ArrayMappingFactory<>(new BookingItemCategory.CategoryObjectMappingFactory()).instantiate(jsonObject.getJSONArray("subCategories").toString());
+                List<String> categoriesIds = new StringArrayMappingFactory().instantiate(jsonObject.getJSONArray("subCategories").toString());
```

Comment:
BookingItemSearchParameters is being used in both QueryItemListMappingFactory and BookingItemSearchResult. are you sure you want this change in both of them ?

Reply:
Yes, I checked. 
In both BookingItemSearchParameters and QueryItemListMappingFactory subCategories come from back-end as array of string ids